features/guidance/collapse.feature

```diff
@@ -143,37 +143,33 @@ Feature: Collapse
     Scenario: Partly Segregated Intersection, Two Segregated Roads
         Given the node map
             """
-              n   m
-
-
-
-
-
-              g   h
-
-
-            c   b   a
-            d   e   f
-
-
-              j   i
-
-
-
-
-
-              k   l
+               n m
+               | |
+               | |
+               | |
+               | |
+               | |
+               g h
+            c - b - a
+            d - e - f
+               j i
+               | |
+               | |
+               | |
+               | |
+               | |
+               k l
```

Comment:
This all depends on which detector is used. Before I only checked the angles between the splitting roads. It turns out that there are a lot of situations, though, with a lot of space between the two segments. If there is a big park in between, I don't think we should completely omit the fact that the road separates into a side left/right of the park. 

To pass the width test between the roads and to make the test check the original behaviour, it was needed to adjust the width between roads here and in some other cases.
The number of lanes is used to calculate the road width. Thats why I had to add it in some places. (think half of the number of lanes times 3 -> 6 roads = 9 m left/right of the road).

It is, of course, not a well distributed feature. Using the lane width shifts the problem of merging/not merging into the data-space rather than the OSRM space. We cannot detect everything validly, so I feel using the data as a method to tune the detection is the best way to go.